pallets/identity/src/lib.rs

```diff
@@ -783,6 +783,39 @@ pub mod migration {
         }
     }
 
+    mod v2 {
+        use super::*;
+        use scale_info::TypeInfo;
+
+        // Authorization struct
+        #[derive(Encode, Decode, TypeInfo, Clone, PartialEq, Debug)]
+        pub struct Authorization<AccountId, Moment> {
+            /// Enum that contains authorization type and data
+            pub authorization_data: AuthorizationData<AccountId>,
+
+            /// Identity of the organization/individual that added this authorization
+            pub authorized_by: IdentityId,
+
+            /// time when this authorization expires. optional.
+            pub expiry: Option<Moment>,
+
+            /// Authorization id of this authorization
+            pub auth_id: u64,
+        }
+
+        decl_storage! {
+            trait Store for Module<T: Config> as Identity {
+                /// All authorizations that an identity/key has
+                pub Authorizations get(fn authorizations): double_map hasher(blake2_128_concat)
+                Signatory<T::AccountId>, hasher(twox_64_concat) u64 => Option<Authorization<T::AccountId, T::Moment>>;
+                    }
```

Comment:
It seems `cargo fmt` can't format inside the `decl_storage` macros.

Please fix the indent here.
```suggestion
            }
```